Setup.bat

```diff
@@ -181,3 +185,12 @@ exit /b 0
 :MarkEndOfBlock
 echo Finished: %~1
 exit /b 0
+
+:ExecuteAndCheck
+%*
+if %ERRORLEVEL% neq 0 (

```

Comment:
Okay, I know this will sound weird, but also try executing Setup.bat by double-clicking on it in the explorer, as that for some reason changes behaviour with respect to error levels, and we had a bug with that previously.
Just want to double-check that it will work like that as well.

Reply:
e.g. this is in the InstallGDK.bat script: https://github.com/improbableio/UnrealEngine/blob/4.25-SpatialOSUnrealGDK-release/InstallGDK.bat#L55

Reply:
Also, maybe we should just rewrite this in powershell.... we call powershell in this batch script to extract the archives, so it's already required, and it would let us deal with these things in a much nicer way.... but probably a separate ticket.

Reply:
I'm able to run this successfully both by clicking from explorer and calling it from cmd

Reply:
What is the behaviour change you are seeing?

Reply:
It was a despair-filled day, and I do not remember much, except Raymond from Microsoft https://devblogs.microsoft.com/oldnewthing/20080926-00/?p=20743, some delayed expansion shenanigans and so on.
It manifested as things returning errorlevel 1 even if they succeeded.

Reply:
Can confirm both running this via cmd and clicking in explorer catches errors